studio/src/pages/RequestorPage/NavigationPanel/RoutesPanel/RoutesPanel.tsx

```diff
@@ -163,24 +174,41 @@ export function RoutesPanel() {
   );
 
   return (
-    <div className={cn("h-full", "flex", "flex-col")}>
+    <>
       <div>
         <div className="flex items-center space-x-2 pb-3">
-          <Input
-            ref={searchRef}
-            className="text-sm"
-            placeholder="Search (hit / to focus)"
-            value={filterValue}
-            onChange={(e) => setFilterValue(e.target.value)}
-            onFocus={() => setSelectedRouteIndex(null)}
-          />
+          <div className="relative flex-grow">
+            <Input
+              ref={searchRef}
+              className={cn(
+                "text-sm",
+                isFocused || filterValue ? "pl-2" : "pl-24",
```

Comment:
I would love it if this was done using `focus:`   This is the only place relying on  `isFocused` right?